src/express_middleware.coffee

```diff
@@ -0,0 +1,19 @@
+module.exports = class ExpressMiddleware
+  constructor: (@rateLimiter) ->
+
+  extractIpsFromReq: (req) =>
```

Comment:
I suppose this isn't entirely necessary as there is no `this` access in the function body, but the reason for it is because this function is used to default the `extractIps` argument in other functions, so it seemed nice to assign the function bound to the proper context.